components/time_slider.py

```diff
@@ -21,9 +21,12 @@ def get_component(min_time, max_time, time_step):
     max_time -- A datetime object representing the end of the time frame
     time_step -- The time step as seconds. One slider step represents
         a time window of this size.
+    interval_seconds -- The update frequency of the time slider in seconds
     """
     seconds = (max_time - min_time).total_seconds()
     steps = ceil(seconds / time_step)
+    if interval_seconds is None:
+        interval_seconds = 2
```

Comment:
This is unnecessary as you give a default value to the parameter.

Reply:
This is in case the user empties the input, in which case a `None` value is passed forward overriding the default parameter value

Reply:
Ok I see. But isn't the default parameter value unnecessary then?

Reply:
Good point! Yes, it is

Reply:
I changed the default value to `None`, as the component is also created before the state of the input is checked.